.travis.yml

```diff
@@ -13,8 +13,9 @@ script:
   - make test
   - bundle exec codeclimate-test-reporter
 env:
-  - RACK_ENV=test
-  - RAILS_ENV=test
+- RACK_ENV=test
+- RAILS_ENV=test
+- COVERAGE=true
```

Comment:
You're right! I missed that. According to the docs, those ENV vars are already set by default: https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables